services/ui_backend_service/data/db/tables/artifact.py

```diff
@@ -0,0 +1,67 @@
+from .base import AsyncPostgresTable
+from .task import AsyncTaskTablePostgres
+from ..models import ArtifactRow
+from services.data.db_utils import translate_run_key, translate_task_key
+# use schema constants from the .data module to keep things consistent
+from services.data.postgres_async_db import AsyncArtifactTablePostgres as MetadataArtifactTable
+import json
+
+
+class AsyncArtifactTablePostgres(AsyncPostgresTable):
+    _row_type = ArtifactRow
+    table_name = MetadataArtifactTable.table_name
+    task_table_name = AsyncTaskTablePostgres.table_name
+    ordering = ["attempt_id DESC"]
+    keys = MetadataArtifactTable.keys
+    primary_keys = MetadataArtifactTable.primary_keys
+    trigger_keys = MetadataArtifactTable.trigger_keys
+    select_columns = keys
+    _command = MetadataArtifactTable._command
+
+    async def get_artifacts_in_runs(self, flow_id: str, run_id: int):
```

Comment:
Should we get rid of these, since most of the UI service relies on `find_records`

Reply:
This comment applies to most of the `tables/*.py` `get_*` functions

Reply:
very true, will clean these up.